### ISSUE TITLE
Remove vox ops - 5 minute rounds are bad

### ISSUE THREAD

**Issue by TheFriggenSawbones** [d3athrow/vgstation13#14458]
http://i.imgur.com/HDZIuMO.png

please god I don't give a shit about some shitty poll you guys had no one enjoys this shit, no one in ooc thinks its a great idea, no one in discord thinks its a good idea.

**Comment by Wizbing** [comment]
My name is Van.
 I'm an artist.
A performance artist.

**Comment by coldcola** [comment]
"please god I don't give a shit about some shitty poll you guys had"
 i feel like i'm tasting delicious ironing

**Comment by Probe1** [comment]
Alright lets remove the round end trigger.

😄

**Comment by gbasood** [comment]
http://ss13.moe/index.php/poll/82

@TheFriggenSawbones you could make a *new* poll but it's only been two months
discord is not our entire playerbase
neither is github

**Comment by ghost** [comment]
Alright, here's how we do it

>Remove the forced round end trigger
>Vox can't go back to their deep space outpost
>Instead, it's more of a regular annoyance
>Teleport things to skrekcomm via an old fashioned teleporter on the ship, receive chiki in return
>chiki can be used to purchase things from skrekcomm, list updates every so often with new things, or changed prices, or things gone out of stock
>Round doesn't end until the station says fuck it and calls the shuttle

**Comment by gbasood** [comment]
@Skullyton good luck assigning values to all teh shit the vox can steal

**Comment by ghost** [comment]
@Skullyton nay on the going to deep space. they should have to go to deep space to return their shit.

i think one of the biggest issues is the boring as fuck objectives. kidnapping objectives need to be MUCH more common

**Comment by Lokiamis** [comment]
I like the suggestion where Vox just continue getting objectives as they complete them.

**Comment by ghost** [comment]
>You mine ore? 1 chiki!
>Oh, you get shiny human thing? 5 chiki!
>You kidnap our target? Excellent, 10 chiki!

Each completed objective yields yet another objective

**Comment by TheFriggenSawbones** [comment]
@gbasood ooc seemed pretty upset at it, after the round ended.

It's getting a negative reaction in a ton of places. And since no one wants to fix it, removing it from rotation seems like an ok idea.

Skullytons idea is a step in the right direction, however.

**Comment by Probe1** [comment]
We just talked about it a few weeks ago.  We're not removing it over one round being upset.  We're continuing to improve it.  

A start is why they weren't required to remove organs.

**Comment by gbasood** [comment]
@Skullyton so they're miners but in a space ship because of  that ore mechanic
it'd probably be a lot more profitable for them to hang out on the asteroid that way
otherwise eehhhh
it might be done but good luck finding a good balance
@TheFriggenSawbones OOC is always upset after a round end and not everyone who's upset in OOC is representative of the entire playerbase

**Comment by TheFriggenSawbones** [comment]
its every time vox raiders comes up, not one round.

**Comment by gbasood** [comment]
do the poll then my dude

**Comment by ghost** [comment]
We need to remove all ways for the round to end 'unnaturally'

Nuke blows up? Yeah that's a round ender
Blob consumed the station? Yeah that's a goner 
Escape shuttle been and gone? Another  station to leave drifting

Nuke ops team die? Shuttle isn't called? Carry on as normal
Vox ops say fuck it and try to go home? Naah fuck off birbs
Raging mages came and gone? Well okay time to tend to my garden

Basically, if it doesn't destroy the station or leave it unoccupied, it shouldn't end the round

**Comment by gbasood** [comment]
yeah that's gonna be after antag datums though

**Comment by kernikov** [comment]
STOP MAKING DUPLICATE ISSUES.
#14252 
#12112
THIS IS 3 NOW FOR YOU ALONE.
HEIST SUCKS.

**Comment by coldcola** [comment]
> another right off

oh jesus.

>raging mages come and gone?

someone's forgetting we have many roundtypes that can go on indefinitely, and are painful as such, thank god we have a few roundtypes that end quickly when the objectives are complete.

**Comment by gbasood** [comment]
given that sawbones made two issues for this already im closing this one
